src/components/MemberProfileCard/memberprofile.module.css

```diff
@@ -2,7 +2,10 @@
   display: flex;
   flex-direction: column;
   padding: 10px 20px;
+  height: 360px;
   width: 30%;
+  position: sticky;
```

Comment:
We wanted to have Fixed Positioning because a fixed component stays at the same position relative to the viewport (the visible area of the browser window) regardless of how much the user scrolls.